tests/test_property.py

```diff
@@ -874,3 +875,479 @@ def test_set_support_mesh(example_model_and_crs):
 
     pc = rqp.PropertyCollection()
     pc.set_support(support = support)
```

Comment:
Could be worth adding an assertion that the support uiud associated with pc does indeed match support.uuid

Reply:
Updated in latest commit